CLI/os_validate.c

```diff
@@ -104,12 +104,12 @@ static int decode_mounts(struct os_configuration *conf)
         char buf[BUFSIZ];
         while (fgets(buf, BUFSIZ, mounts) != NULL) {
             char *str = &buf[0];
-            char *device = strsep(&str, " ");
+            strsep(&str, " "); //skip device
```

Comment:
We decided to use C-style comments, so please make this modification:
```suggestion
            strsep(&str, " "); /* skip device */
```

Reply:
Thanks!